### ISSUE TITLE
Ability to add custom connectors like Torus

### ISSUE THREAD

**Issue by xpluscal** [softwarecurator/svelte-wagmi#3]
Hey! You're a lifesaver. This package is amazingly helpful (!!!)

I'd love to get more access to the createClient from wagmi, e.g. to add multiple chains and connectors.
What's the best way to add e.g. Torus connector in there?

https://github.com/torusresearch/torus-wagmi-connector

Thank you!

**Comment by softwarecurator** [comment]
will look into this it would be nice to allow more connectors for other use cases

**Comment by xpluscal** [comment]
yeah that would be awesome. I actually created a fork where the connectors are just passed as an optional array and then spread into your connectors at createClient.

Regarding Torus: Unfortunately, I discovered that they didn't update the package to be compatible with wagmi > 1.0.
(It's still built on 0.5.x which means it's based on ethers...

But there are certainly other connectors like web3auth that are interesting.
